robot/client/client_options.go

```diff
@@ -15,12 +15,14 @@ type robotClientOpts struct {
 	refreshEvery *time.Duration
 
 	// checkConnectedEvery is how often to check connection to the
-	// robot. If unset, it will not be checked automatically.
-	checkConnectedEvery time.Duration
+	// robot. If <0, it will not be refreshed automatically, if unset,
+	// it will automatically refresh every 10s
+	checkConnectedEvery *time.Duration
 
 	// reconnectEvery is how often to try reconnecting the
-	// robot. If unset, it will not be reconnected automatically.
-	reconnectEvery time.Duration
+	// robot. If <0, it will not be refreshed automatically, if unset,
```

Comment:
<=

Reply:
fixed

Reply:
can you also update the < for refreshEvery?